src/main/java/com/google/cloud/tools/appengine/cloudsdk/PathResolver.java

```diff
@@ -50,6 +50,8 @@ public Path getCloudSdkPath() {
     } else {
       // home directory
       possiblePaths.add(System.getProperty("user.home") + "/google-cloud-sdk");
+      // usr directory
+      possiblePaths.add("/usr");
```

Comment:
I think what you want here is /usr/bin, not /usr. And that said what we need here is the path to the actual google-cloud-sdk directory such as /usr/local/google-cloud-sdk, not simply a path to symlinks the installer may set up elsewhere. These paths are used for more than just the binaries. 

I don't have a Debian system handy to test this with. Where is the installer putting the google-cloud-sdk directory on your system?